src/database/constants.ts

```diff
@@ -19,4 +20,5 @@ export enum Config {
   ROLES = 'roles',
   MAIL = 'mail',
   BIRTHDAYS = 'birthdays',
+  COUNTDOWN = 'countdown',
```

Comment:
Is this intentionally singular, unlike `BIRTHDAYS`?

Reply:
Good catch, making it plural